test/test.js

```diff
@@ -0,0 +1,608 @@
+let fs = require('fs')
```

Comment:
We don't need a separate folder only for this one test file. Any reason for that?

Reply:
No special reason, moved the `test.js` file back to top level